relnotes/scheduler-exceptions.migration.md

```diff
@@ -0,0 +1,21 @@
+## New default scheduler exception handler in tests
```

Comment:
This breaking change is slipped in because you're already making an important breaking change a few commits later, right?

Should the policy rather be to absolutely minimise breaking changes in minor branches? (Rather than a kind of free-for-all attitude, once one breaking change has been included ;)

Reply:
No, this one is actually quite related. With old default handler migrating main breaking change is much harder because exception from failed `verify` in `Task.resume` could be lost inside epoll or replaced by some other exception during the unwinding, making it much harder to debug any test failure caused by it.

Thus this change is to actually make another breaking change less annoying.

Reply:
Ok :+1: